ast_comments.py

```diff
@@ -8,6 +8,9 @@
 
 
 class Comment(ast.AST):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("op", "values")
```

Comment:
I don't understand the match-args-part. Why `("op", "values")`?
`Comment` does not define these attributes. It works with `value` and `inline` instead.

```suggestion
        __match_args__ = ("value", "inline")
```